styleguide/assets/variables.css

```diff
@@ -1,54 +1,250 @@
+}
+
+/* Light */
+@media (prefers-color-scheme: light) {
+	:root {
+		--color-main-background: #ffffff;
+		--color-main-background-rgb: 255, 255, 255;
+		--color-main-background-translucent: rgba(
+			var(--color-main-background-rgb),
+			0.97
+		);
+		--color-main-background-blur: rgba(
+			var(--color-main-background-rgb),
+			0.8
+		);
+		--filter-background-blur: blur(25px);
+		--gradient-main-background: var(--color-main-background) 0%,
+			var(--color-main-background-translucent) 85%, transparent 100%;
+		--color-background-hover: #f5f5f5;
+		--color-background-dark: #ededed;
+		--color-background-darker: #dbdbdb;
+		--color-placeholder-light: #e6e6e6;
+		--color-placeholder-dark: #cccccc;
+		--color-primary: #0082c9;
+		--color-primary-text: #ffffff;
+		--color-primary-hover: #329bd3;
+		--color-primary-light: #e5f2f9;
+		--color-primary-light-text: #0082c9;
+		--color-primary-light-hover: #dbe7ee;
+		--color-primary-text-dark: #ededed;
+		--color-primary-element: #0082c9;
+		--color-primary-element-text: #ffffff;
+		--color-primary-element-hover: #329bd3;
+		--color-primary-element-light: #e5f2f9;
+		--color-primary-element-light-text: #0082c9;
+		--color-primary-element-light-hover: #dbe7ee;
+		--color-primary-element-text-dark: #ededed;
+		--gradient-primary-background: linear-gradient(
+			40deg,
+			var(--color-primary) 0%,
+			var(--color-primary-hover) 100%
+		);
+		--color-main-text: #222222;
+		--color-text-maxcontrast: #767676;
+		--color-text-light: #222222;
+		--color-text-lighter: #767676;
+		--color-error: #e9322d;
+		--color-error-rgb: 233, 50, 45;
+		--color-error-hover: #ed5a56;
+		--color-warning: #eca700;
+		--color-warning-rgb: 236, 167, 0;
+		--color-warning-hover: #efb832;
+		--color-success: #46ba61;
+		--color-success-rgb: 70, 186, 97;
+		--color-success-hover: #6ac780;
+		--color-loading-light: #cccccc;
+		--color-loading-dark: #444444;
+		--color-box-shadow-rgb: 77, 77, 77;
+		--color-box-shadow: rgba(var(--color-box-shadow-rgb), 0.5);
+		--color-border: #ededed;
+		--color-border-dark: #dbdbdb;
+		--font-face: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+			Oxygen-Sans, Cantarell, Ubuntu, "Helvetica Neue", Arial, sans-serif,
+			"Noto Color Emoji", "Apple Color Emoji", "Segoe UI Emoji",
+			"Segoe UI Symbol";
+		--default-font-size: 15px;
+		--animation-quick: 100ms;
+		--animation-slow: 300ms;
+		--border-radius: 3px;
+		--border-radius-large: 10px;
+		--border-radius-pill: 100px;
+		--default-line-height: 24px;
+		--header-height: 50px;
+		--navigation-width: 300px;
+		--sidebar-min-width: 300px;
+		--sidebar-max-width: 500px;
+		--list-min-width: 200px;
+		--list-max-width: 300px;
+		--header-menu-item-height: 44px;
+		--header-menu-profile-item-height: 66px;
+		--breakpoint-mobile: 1024px;
+		--primary-invert-if-bright: no;
+		--background-invert-if-dark: no;
+		--background-invert-if-bright: invert(100%);
+	}
+}
+
+/* Dark */
+@media (prefers-color-scheme: dark) {
```

Comment:
I would skip the dark color scheme and only update the light one. Otherwise the rest of the styleguide would need adjustments as well, because it looks weird at the moment. Maybe something for a follow-up to have dark mode supported in the styleguide.

Reply:
Agreed, we can add the dark one at some point if the style guide supports it.

Reply:
I believe this is just exposing a problem we already have.
This css is from server, if it looks weird in styleguide, why not is look weird in nextcloud?

Reply:
Because the styleguide is not Nectcloud server. It does not bring all styles the server does and components are used without the correct containers there, i.e. an app must use NcAppContent as the root container, which the styleguide obviously does not.

Reply:
I've removed the dark and light entries.
We can discuss it better later.